src/pages/index.tsx

```diff
@@ -105,18 +117,22 @@ const Index: FC = () => {
       <Head title='Tasks' />
         <TasksProvider >
           <div className={classNames.container}>
-            {!!error && <p>Something went wrong, please contact admin!</p>}
+            {!!error && <p>{TASKS_FETCH_ERROR_MESSAGE}</p>}
             {isLoading ? (
               <p>Loading...</p>
             ) : (
               <>
                 {Object.keys(filteredTask).length > 0
-                  ? Object.keys(filteredTask).map((key) => (
-                    <Accordion open={(statusActiveList.includes(key))} title={key} key={key}>
-                      <TaskList tasks={filteredTask[key]} isEditable={isEditable} updateCardContent={updateCardContent} hasLimit={key == IN_PROGRESS}/>
-                    </Accordion>
-                  ))
-                  : !error && 'No Tasks Found'}
+                  ? <div className={classNames.tasksContainer}>
+                      <div className={classNames.tabsContainer}>
+                        <Tabs tabs={Object.values(Tab) as Tab[]} onSelect={onSelect} activeTab={activeTab}/>
+                      </div>
+                      <div>
+                        {filteredTask[activeTab] ? <TaskList tasks={filteredTask[activeTab]} isEditable={isEditable} updateCardContent={updateCardContent}/>
+                        :<EmptyTaskListLabel/>}
+                      </div>
+                    </div>
+                  : !error && <EmptyTaskListLabel/>}
```

Comment:
The highlighted piece of code has multiple levels of ternary operators being used which is making this code hard to understand/debug. I'd suggest looking for ways to flatten it out OR move some parts of code into separate functions.

```typescript
const renderTabSection = () => (
  <div className={classNames.tabsContainer}>
    <Tabs
      tabs={Object.values(Tab) as Tab[]}
      onSelect={onSelect}
      activeTab={activeTab}
    />
  </div>
);

const renderTaskList = () => (
  <div>
    {filteredTask[activeTab] ? (
      <TaskList
        tasks={filteredTask[activeTab]}
        isEditable={isEditable}
        updateCardContent={updateCardContent}
      />
    ) : (
      <EmptyTaskListLabel />
    )}
  </div>
);

.
.
.
  <div className={classNames.tasksContainer}>
    {renderTabSection()}
    {renderTaskList()}
  </div>
.
.
.

```